src/Gauge/ThermalAssistantRenderer.cpp

```diff
@@ -127,6 +127,14 @@ ThermalAssistantRenderer::PaintRadarPlane(Canvas &canvas, double max_lift) const
                        +Layout::FastScale(small ? 2 : 4)),
                   p.At(-Layout::FastScale(small ? 2 : 4),
                        +Layout::FastScale(small ? 2 : 4)));
+
+  if (small)
+    return;
+
+  if (!circling.TurningLeft()) 
+    DrawCircleLabelVSpeed(canvas, p + PixelSize{-45, 0}, vario.average);
+  else 
+    DrawCircleLabelVSpeed(canvas, p + PixelSize{+45, 0}, vario.average);
```

Comment:
These two lines are duplicate, the only difference is the sign. This should be parametrized. Also where does this `45` come from? Don't hard-code pixel sizes!